app/views/backend/course.erb

```diff
@@ -15,6 +15,7 @@
                 <% @course.chapters.each do |chapter| %>
                    <li>
                     <%= chapter.name %>
+                    <a href="/chapter/delete/<%= chapter.id %>" <i class="material-icons">delete</i></a><br>
```

Comment:
but the <ul> shouldn't be about chapters? if we put each element inside a <li> when we have a course with 10 chapters we won't be able to tell wich chapter name or description belongs to each chapter.